scratch/get_config_for_failed_postprocess.R

```diff
@@ -31,5 +31,5 @@ new_eval_config$forecast_date_hosp <- "2023-10-16"
 
 yaml::write_yaml(new_eval_config, file = file.path(
   config_dir,
-  glue::glue("eval_config_rerun.yaml")
+  glue::glue("eval_config.yaml")
```

Comment:
1. Yes. The rerun probably shouldn't have bin -- you have to create a new `eval_config.yaml` to rerun the failed jobs in azure since it expects `eval_config.yaml`. This is in scratch so I do think its fine, and this is just an issue that is an artifact of the azure jobs failing post processing bc of memory issues at states with large number of sites. I think I renamed them to toggle back and forth